spec/factories/gift_card_orders.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :gift_card_order do
+    user { nil }
+    gift_card { nil }
+    status { "MyString" }
```

Comment:
This isn't correctly set up.

```suggestion
    user
    gift_card { GiftCard.first }
    status { :completed }
```